Traceback (most recent call last):
  File "/Applications/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd.py", line 1491, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Applications/PyCharm.app/Contents/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/ming/Documents/TieComm/main.py", line 176, in <module>
    main(args)
  File "/Users/ming/Documents/TieComm/main.py", line 65, in main
    env = env_REGISTRY[args.env_args['env']](args.env_args)
TypeError: env_fn() got multiple values for argument 'env'